modules/stanford_courses_tag_translate/stanford_courses_tag_translate.install

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * @file
+ */
+
+/**
+ * Implements hook_install().
+ *
+ * Look up wether or not the feed importer we want to modify is in the database.
+ * If the settings are in the datbase then we need to alter it and save it back.
+ * If the settings are in code do nothing and let the alters to their work.
+ *
+ */
+function stanford_courses_tag_translate_install() {
+
+  ctools_include('export');
+
+  // Check to see if the importer has been modified and is in the database.
+  $importers = ctools_export_load_object('feeds_importer', 'conditions', array('id' => 'stanford_course_importer'));
+
+  // Does not exist. This is strange but end anyway. Should we log here?
```

Comment:
Hrm, good question. Without doing a lot of digging, I would assume that the `$importers["stanford_course_importer"]` object would at least contain some metadata, therefore not be empty. I think I'm fine with this just being addressed as a code comment at this point.